<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-28T16:36:07 IST" name="SerialSuite" finished-at="2025-09-28T16:36:38 IST" duration-ms="31142">
    <groups>
    </groups>
    <test started-at="2025-09-28T16:36:07 IST" name="Test" finished-at="2025-09-28T16:36:38 IST" duration-ms="31142">
      <class name="com.example.automation.tests.EndToEndTests">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.example.automation.tests.EndToEndTests@2925bf5b]" started-at="2025-09-28T16:36:07 IST" name="setup" finished-at="2025-09-28T16:36:14 IST" duration-ms="6863" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="endToEndTest()[pri:0, instance:com.example.automation.tests.EndToEndTests@2925bf5b]" started-at="2025-09-28T16:36:14 IST" name="endToEndTest" finished-at="2025-09-28T16:36:31 IST" duration-ms="16703" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at com.example.automation.tests.EndToEndTests.endToEndTest(EndToEndTests.java:54)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endToEndTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.example.automation.tests.EndToEndTests@2925bf5b]" started-at="2025-09-28T16:36:33 IST" name="tearDown" finished-at="2025-09-28T16:36:34 IST" duration-ms="391" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.example.automation.tests.EndToEndTests -->
      <class name="com.example.automation.tests.VerifyStatusAPI">
        <test-method signature="verifyStatusAPI()[pri:0, instance:com.example.automation.tests.VerifyStatusAPI@510f3d34]" started-at="2025-09-28T16:36:34 IST" name="verifyStatusAPI" finished-at="2025-09-28T16:36:38 IST" duration-ms="4240" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyStatusAPI -->
      </class> <!-- com.example.automation.tests.VerifyStatusAPI -->
    </test> <!-- Test -->
  </suite> <!-- SerialSuite -->
</testng-results>
